products/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db   import models
+
+from users.models import Host
+from core.models import TimeStampModel
+
+class Category(TimeStampModel): 
+    name = models.CharField(max_length=200)
+
+    class Meta: 
+        db_table = 'categories'
+
+class Subcategory(TimeStampModel): 
+    category = models.ForeignKey('Category', on_delete=models.CASCADE, related_name='categories')
```

Comment:
related_name `sub_categories`로 만들어야 하지 않나요?

```suggestion
    category = models.ForeignKey('Category', on_delete=models.CASCADE, related_name='categories')
```

Reply:
넵! 변경했습니다~~